### ISSUE TITLE
WCAG 2.5.2 (Level A) : Actions must not trigger on press or touch down

### ISSUE THREAD

**Issue by markfullmer** [ckeditor/ckeditor5#19347]
## 📝 Provide detailed reproduction steps (if any)

1. …Activate any CKEditor element that has a "balloon" interface.
2. …Click anywhere outside of the balloon interface to dismiss it
3. …The balloon interface is dismissed on key/press down, rather than mouse up or finger-lift.

### ✔️ Expected result

The balloon interface is dismissed on mouse up or finger-lift.

### ❌ Actual result

The balloon interface is dismissed on key/press down, rather than mouse up or finger-lift.

### Background
This constitutes a failure a Level A Success Criterion as documented in https://www.w3.org/WAI/WCAG22/Understanding/pointer-cancellation.html

The specific failure is documented in https://www.w3.org/WAI/WCAG22/Techniques/failures/F101 :

> The objective of this Failure is to describe situations where:
> 
>     Controls are set to activate functionality on the down-event of a pointer;
>     No further mechanism to abort or undo is available;
>     The up-event does not reverse the outcome of the activation;
>     It is not essential for the functionality to execute and complete on the down-event.

### Proposed resolution
Change the behavior so that balloons are dismissed on the "up" event, rather than the "down" event.
